packages/cojson-transport-ws/src/index.ts

```diff
@@ -43,26 +40,20 @@ export function createWebSocketPeer({
 
     let pingTimeout: ReturnType<typeof setTimeout> | null = null;
 
+    let supportsBatching = batchingByDefault;
+
     websocket.addEventListener("message", function handleIncomingMsg(event) {
-        const msg = JSON.parse(event.data as string);
-        pingTimeout && clearTimeout(pingTimeout);
-
-        if (msg?.type === "ping") {
-            const ping = msg as PingMsg;
-            g.jazzPings ||= [];
-            g.jazzPings.push({
-                received: Date.now(),
-                sent: ping.time,
-                dc: ping.dc,
-            });
```

Comment:
This code as it seemed unused, happy to restore it if we think that could be useful.

Reply:
We'll replace it with a better latency measurement system in the future